polaris/utils/progress_indicator.py

```diff
@@ -0,0 +1,32 @@
+from contextlib import contextmanager
+from yaspin import yaspin
+from yaspin.spinners import Spinners
+
+
+@contextmanager
+def progress_indicator(start_msg: str = "In progress...", success_msg: str = "", error_msg: str = ""):
```

Comment:
Would it make sense to have sensible defaults here (as opposed to empty strings)? Just something simple like `success_msg="Success", error_msg="Error"`?

Reply:
I put them as empty strings because the `spinner.ok` and `spinner.fail` methods already print out `SUCCESS` and `FAIL`. 

I think it might actually be better to not make these arguments optional as every call of `progress_indicator` in the client defines them (and they should for clarity).